include/muu/integer.h

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+//
+
+#pragma once
+
+//! \file
+//! \brief Shorthands for fixed width integer types.
```

Comment:
I use the `///` style for docstrings throughout the codebase.
```suggestion
/// \file
/// \brief Shorthands for fixed width integer types.
```

Reply:
Oh wow, that slipped in, I thought I had already changed it